src/patternfly/components/Content/content.scss

```diff
@@ -1,6 +1,6 @@
 @use '../../sass-utilities' as *;
 
-@include pf-root($content) {
+@include pf-root(':root', false) {
```

Comment:
Since we support `.pf-v6-c-content--[all content elements]` as a standalone class, outside of `.pf-v6-c-content` now, these vars need to be accessible outside of `.pf-v6-c-content`. Originally I had changed this to `[class*="pf-v6-c-content"]` but that won't work with user overrides since we'll be resetting the variables for each of those classes. So maybe content can be an exception and continue to scope its vars to `:root`?